amazon-msk/cdk/web-streaming/cdk.json

```
@@ -23,10 +23,14 @@
       "msk":
       {
         "bootstrapServers": "<Bootstrap Servers of your MSK cluster>",
-        "credentialsSecretName": "<Secret Name associated with your MSK cluster>"  
+        "credentials": "<Secret Name associated with your MSK cluster>"  
       },
-      "publicTlsCertificateKey": "<your public tls certificate key ARN>",
-      "kafkaTopic": "<your kafka topic to be exposed on HTTP and SSE>"
+      "public":
+      {
+        "tlsCertificateKey": "<your public tls certificate key ARN>",
```

```suggestion
        "certificate": "<your public tls certificate key ARN>",
```
This reads better I think, and the value comment seems sufficient for clarity too, agree?